rootfs/etc/services.d/autodiscripper/run

```diff
@@ -177,7 +181,7 @@ while true; do
     rm "$DRIVES_INFO"
     [ "$FIRST_RUN" -eq 0 ] || log "Ready."
     FIRST_RUN=0
-    sleep 5
+    sleep $AUTO_DISC_RIPPER_INTERVAL
```

Comment:
Thanks for the pull request!
Since the new variable is used only once, we can just do:
```
sleep "${AUTO_DISC_RIPPER_INTERVAL:-5}"
```
No need to define the variable explicitly before the loop.